extensions/wechat-devtool/src/utils/command.ts

```diff
@@ -40,6 +34,35 @@ export async function openProject(cliPath: string, projectPath: string) {
   throw new Error(stderr || stdout);
 }
 
+export async function previewProject(cliPath: string, projectPath: string, projectId: string) {
+  await mkdir(PREVIEW_QRCODE_DIR, { recursive: true });
+
+  const qrcodePath = path.resolve(PREVIEW_QRCODE_DIR, `${projectId}.png`);
+
+  const command = `${cliPath} preview --project ${projectPath} --qr-size small --qr-format image --qr-output ${qrcodePath}`;
```

Comment:
The project path may contain spaces. You might need to quote them or use `execFile` instead.

Reply:
thx 👍